dev/com.ibm.ws.security.javaeesec.cdi/resources/com/ibm/ws/security/javaeesec/cdi/internal/resources/JavaEESecMessages.nlsprops

```diff
@@ -41,3 +41,11 @@ JAVAEESEC_CDI_ERROR_UNIQUE_ID_NULL.useraction=If the application provides an Ide
 JAVAEESEC_CDI_WARNING_NO_IDENTITY_STORE_HANDLER=CWWKS1930W: IdentityStoreHandler bean is not found. Use a User Registry which is defined by server.xml. IdentityStoreHandler bean is created when there is at least one IdentityStore bean. If the application intends to use an IdentityStore, make sure that it is configured properly. 
 JAVAEESEC_CDI_WARNING_NO_IDENTITY_STORE_HANDLER.explanation=IdentityStoreHandler bean is not found. If the application intends to use an IdentityStore, make sure that it is configured properly. IdentityStoreHandler bean is created when there is at least one IdentityStore bean.
 JAVAEESEC_CDI_WARNING_NO_IDENTITY_STORE_HANDLER.useraction=If the application intends to use an IdentityStore, make sure that it is configured properly.
+
+JAVAEESEC_CDI_ERROR_LOGIN_CONFIG_EXISTS=CWWKS1931E: The application deployment failed because login-config element is found in web.xml of application {0} module {1}. Make sure removing login-config element from web.xml in order to enable HttpAuthenticationMechanism annotation.
+JAVAEESEC_CDI_ERROR_LOGIN_CONFIG_EXISTS.explanation=The application deployment failed because login-config element is found in web.xml. This is an error condition.
+JAVAEESEC_CDI_ERROR_LOGIN_CONFIG_EXISTS.useraction=Make sure removing login-config element from web.xml in order to enable HttpAuthenticationMechanism annotation.
```

Comment:
Edits below:
JAVAEESEC_CDI_ERROR_LOGIN_CONFIG_EXISTS=CWWKS1931E: The application deployment failed because a login-config element was found in the web.xml file for the {1} module in the {0} application. To enable the HttpAuthenticationMechanism annotation, remove the login-config element from the web.xml file.
**I had to reorder for it to make more sense.**
JAVAEESEC_CDI_ERROR_LOGIN_CONFIG_EXISTS.explanation=The application deployment failed because both the login-config element and the HttpAuthenticationMechanism annotation are defined. Only one of these authentication mechanisms can be used in your application.
**I tried to add more info here - please review for technical accuracy.**
JAVAEESEC_CDI_ERROR_LOGIN_CONFIG_EXISTS.useraction= To enable the HttpAuthenticationMechanism annotation, remove the login-config element from the web.xml file.

Reply:
Got it. I modified the text.